src/components/application_manager/test/resumption/resumption_data_test.cc

```diff
@@ -149,6 +149,7 @@ void ResumptionDataTest::CheckChoiceSet(sm::SmartObject& res_list) {
       sm::SmartObject command = res_list[i][am::strings::choice_set][j];
       EXPECT_EQ(i + j, command[am::strings::choice_id].asUInt());
       char numb[12];
+      // cppcheck-suppress invalidPrintfArgType_sint
       std::snprintf(numb, 12, "%d", i + j);
```

Comment:
@Ypostolov Do not suppress this issue. You can find solution at several comment above.